### PR TITLE
ShortTagFixer - fix priority

### PR DESCRIPTION
**Pull request by keradus**

### CODE REVIEW

**Comment by SpacePossum** [comment]
nice find and fix :)
:+1:

**Comment by gharlan** [comment]
:+1:

**Comment by GrahamCampbell** [comment]
What about 1.11?

**Comment by keradus** [comment]
Yes, it's for 1.11, even named my branch after it :D

**Comment by GrahamCampbell** [comment]
:)

File: Symfony/CS/Tests/Fixtures/Integration/misc/php_closing_tag,short_tag.test [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+--REQUIREMENTS--
+hhvm=false
+--INPUT--
+<?
+$a = 1;
+
+?>
+--EXPECT--
+<?php
+$a = 1;
```
Comment (Soullivaneuh) [inline Symfony/CS/Tests/Fixtures/Integration/misc/php_closing_tag,short_tag.test]: Maybe add a new line?

Comment (keradus) [inline Symfony/CS/Tests/Fixtures/Integration/misc/php_closing_tag,short_tag.test]: Then it won't pass.
And I don't want to add extra fixer to this test configuration just to have that line.

**Pull request closed without merging**